data/roles.js

```diff
@@ -852,81 +878,85 @@ const roleData = {
-                "Wins when mafia wins.",
+                "Wins with mafia.",
                 "Does not count towards mafia win count.",
             ],
         },
         "Mastermind": {
             alignment: "Independent",
             description: [
-                "Mafia meeting is anonymous",
-                "Wins instead of mafia and counts toward their total",
+                "Mafia meeting is anonymous if Mastermind is present in the game.",
+                "Wins instead of mafia and counts toward their total.",
                 ],
         },
         "Lover": {
             alignment: "Independent",
             description: [
-                "Once per game can fall in love with another player.",
-                "Both players die if either dies.",
+                "Fall in love with another player, once per game.",
+                "Both players die if either of them is killed.",
                 "Wins if both players survive until the end of the game.",
             ],
         },
         "Matchmaker": {
             alignment: "Independent",
             description: [
-                "Each night chooses two people to go on a date. If those two are the same alignment, they will fall in love.",
+                "Each night chooses two players to go on a date. If they are the same alignment, they will fall in love.",
                 "Wins if all players left alive are in love.",
             ],
         },
         "Prophet": {
             alignment: "Independent",
             description: [
-                "Once per game can predict when the game will end.",
-                "Wins if they guess correctly.",
+                "Predict once per game, the day/night cycle the game will end.",
+                "Wins if guess is correct.",
             ],
         },
         "Vengeful Spirit": {
             alignment: "Independent",
             description: [
-                "If killed by any other players in a way that is not the village vote, will gain the ability to kill a player each night in the graveyard.",
-                "Wins if it kills all of its murderers.",
+                "If murdered by another player, gains the ability to kill each night from the graveyard.",
+                "Does not gain the ability if executed by village vote.",
+                "Wins if they kill all of their murderers.",
             ],
             graveyardParticipation: "self",
         },
         "Clockmaker": {
             alignment: "Independent",
             description: [
-                "Can kill one player each night.",
-                "Has a clock that starts at 6.",
-                "Whenever they kill a player the clock changes based on that player's alignment.",
-                "Goes up by 1 for village, up by 2 for Mafia or Monster, down by 3 for independant.",
-                "Wins when if clock strikes 12, gains an extra life at 9, instantly dies at 3."
+                "Has a clock that starts at 6 PM.",
```

Comment:
Regarding the AM/PM stuff in the clockmaker description, I don't know if it needs to be explicit. I'm not sure, but it might be possible for Clockmaker's clock to bypass 12 and hit 3, and still kill the clockmaker, skipping their win condition of hitting 12.  This is probably fine? But might want to double check this.